core/src/refx/hooks.cljs

```diff
@@ -28,7 +28,7 @@
         (react/useMemo
          (fn []
            [(fn [callback]
-              (let [key (str "use-sub-" (swap! use-sub-counter inc))]
+              (let [key {:index (swap! use-sub-counter inc)}]
```

Comment:
Switched the key so it can be easily sortable since `use-sub-11` would come before `use-sub-2` when using default string comparison.